api.md

```diff
@@ -1685,53 +1686,41 @@ GROUP BY <time_bucket( <const_value>, <partition_col_of_hypertable> ),
 |**Description**|**Type**|**Default**|
-|If timescaledb.continuous is not specified, then this is a regular PostgresSQL view. | `BOOLEAN` ||
+|If timescaledb.continuous is not specified, then this is a regular PostgresSQL materialized view. | `BOOLEAN` ||
 
 #### Optional `WITH` clause options [](continuous_aggregate-create_view-with-optional)
 
 |**Name**|||
 |---|---|---|
-|`timescaledb.refresh_lag`|||
-|**Description**|**Type**|**Default**|
-| Refresh lag controls the amount by which the materialization will lag behind the current time. The continuous aggregate view lags behind by `bucket_width` + `refresh_lag` value. `refresh_lag` can be set to positive and negative values. | Same datatype as the `bucket_width` argument from the `time_bucket` expression.| The default value is twice the bucket width (as specified by the `time_bucket` expression).|
-|   |   |   |
-|**Name**|||
-|`timescaledb.refresh_interval`|||
-|**Description**|**Type**|**Default**|
-| Refresh interval controls how often the background materializer is run. Note that if `refresh_lag` is set to `-<bucket_width>`, the continuous aggregate will run whenever new data is received, regardless of what the `refresh_interval` value is. | `INTERVAL`|By default, this is set to twice the bucket width (if the datatype of the bucket_width argument from the `time_bucket` expression is an `INTERVAL`), otherwise it is set to 12 hours.|
-|   |   |   |
-|**Name**|||
 |`timescaledb.materialized_only`|||
 |**Description**|**Type**|**Default**|
 | Return only materialized data when querying the continuous aggregate view. | `BOOLEAN` | false |
 |   |   |   |
-|**Name**|||
-|`timescaledb.max_interval_per_job`|||
-|**Description**|**Type**|**Default**|
-| Max interval per job specifies the amount of data processed by the background materializer job when the continuous aggregate is updated. | Same datatype as the `bucket_width` argument from the `time_bucket` expression.| The default value is `20 * bucket width`.|
-|   |   |   |
-|**Name**|||
 |`timescaledb.create_group_indexes`|||
 |**Description**|**Type**|**Default**|
 | Create indexes on the materialization table for the group by columns (specified by the `GROUP BY` clause of the `SELECT` query). | `BOOLEAN` | Indexes are created by default for every group by expression + time_bucket expression pair.|
-|   |   |   |
-|**Name**|||
-|`timescaledb.ignore_invalidation_older_than`|||
-|**Description**|**Type**|**Default**|
-| Time interval after which invalidations are ignored.| Same datatype as the `bucket_width` argument from the `time_bucket` expression. | By default all invalidations are processed.|
 
->:TIP: Say, the continuous aggregate uses time_bucket(INTERVAL '2h', time_column) and we want to keep the view up to date with the data. We can do this by modifying the `refresh_lag` setting. Set refresh_lag to `-2h`. E.g. `ALTER VIEW contview set (timescaledb.refresh_lag = '-2h');` Please refer to the [caveats][].
+#### Notes
```

Comment:
@solugebefola Do you have thoughts on any standard template format for API docs with such type of "notes"?